### PR TITLE
POC: Test gobuild cobertura coverage

### PR DESCRIPTION
**Pull request by ichekrygin**
**Description of your changes:**

**Which issue is resolved by this Pull Request:**
Resolves #

**Checklist:**
- [ ] Documentation has been updated, if necessary.
- [ ] Pending release notes updated with breaking and/or notable changes, if necessary.
- [ ] Upgrade from previous release is tested and upgrade user guide is updated, if necessary.
- [ ] Code generation (`make generate`) has been run to update object specifications, if necessary.
- [ ] CRD manifests generation (`make manifests`) has been run to update CRD manifests yaml file specifications, if necessary.
- [ ] Update dependencies (`make vendor`) has been run to update `Gopkg.lock` file, if necessary
- [ ] All related commits have been squashed to improve readability.
- [ ] Comments have been added or updated based on the standards set in [CONTRIBUTING.md](../blob/master/CONTRIBUTING.md#comments)

### CODE REVIEW

File: cmd/crossplane/main.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func main() {
 
 	// Re-sync resources every minutes
 	// TODO: 1 minute could be too aggressive - we will update this in the future.
-	syncPeriod := 1 * time.Minute
+	syncPeriod := 60 * time.Minute
```
Comment (negz) [inline cmd/crossplane/main.go]: Was this supposed to be included in this change?

Comment (ichekrygin) [reply cmd/crossplane/main.go]: Ah.. no, good catch.

**Comment by negz** [comment]
https://github.com/jenkinsci/github-pr-coverage-status-plugin

@ichekrygin The above Jenkins module would get us nice, succinct coverage statuses as Github PR comments and supports Cobertura.

**Comment by ichekrygin** [comment]
@negs, do you prefer that over `codecov` pr comments?

**Comment by negz** [comment]
> do you prefer that over codecov pr comments?

I do not. I thought it might be a decent alternative if you decided to go with a pure Cobertura implementation, but I don't have a preference either way.

**Pull request closed without merging**